google/cloud/firestore_v1/query.py

```diff
@@ -506,3 +566,122 @@ def get_partitions(
             start_at = cursor
 
         yield QueryPartition(self, start_at, None)
+
+
+class Iterator(page_iterator.Iterator):
```

Comment:
Why turn this into a page_iterator? Previously, it just returned a standard python generator

Reply:
I was using datastore as a reference, but apparently as you noted below, I could just use a class to wrap this method.